src/main/java/Duke.java

```diff
@@ -1,10 +1,108 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static final String HORIZONTAL_LINE = "____________________________________________________________\n";
+    private static final String ADD_CONFIRMATION_MESSAGE = "Got it. I've added this task:";
+    private static Task[] inputs = new Task[100];
+    private static int index = 0;
+
+    /**
+     * Returns a string with formatted user inputs.
+     *
+     * @param inputs list of user input.
+     * @return Formatted string to print
+     */
+    public static String formatList(Task[] inputs) {
```

Comment:
No comments for all functions. Can consider adding based on the java coding standards so that its easier to read & understand your code